rllib/algorithms/algorithm_config.py

```diff
@@ -2930,6 +2930,31 @@ def get_marl_module_spec(
                         "is passed in nor in the default module spec used in "
                         "the algorithm."
                     )
+            if module_spec.catalog_class is None:
```

Comment:
Further up is a typo in L2905:

# PolicySpecs and the **defaul** RLModuleSpec.